SQL/Cleanup_patches/README.md

```diff
@@ -0,0 +1,3 @@
+This directory contains SQL patches written as a post-release clean-up. These patches can contain `ALTER TABLE`, `DELETE` and/or `DROP` statements.
+
```

Comment:
Does that mean someone could include the following instruction in such cleanup patches?
ALTER TABLE ADD COLUMN ....

Reply:
@PapillonMcGill they could include whatever they want as long as its not critical to the functioning of the code.

things that go here are things that usually are commented out in the release patches

Reply:
added a line to explain